### DISCUSSION TITLE
How is ApplicationSet Webhook related to ArgoCD Server Webhook?

### DISCUSSION THREAD

**Discussion by jacek-jablonski** [argoproj/argo-cd#11905] [Q&A]
Hi,
I've noticed, that ApplicationSet has its own webhook. How it is related to ArgoCD Server webhook. In GitHub should I have configured both webhooks for Server and ApplicationSet?

**Comment by ptchau2003** [comment]
I'd like to have same question, because the webhook is sending to ApplicationSet is not working for me at the moment (not trigger the ArgoCD sync :( )

**Comment by ptchau2003** [comment]
@jacek-jablonski I guess the answer is
- Argocd Webhook is for application - refer here: https://github.com/argoproj/argo-cd/blob/master/util/webhook/webhook.go
- ApplicationSet Webhook is for ApplicationSet only (in case we need to update the ApplicationSet)
https://github.com/argoproj/argo-cd/tree/master/applicationset/webhook

I made mistake because I am sending the webhook to ApplicationSet and the webhook does not trigger the sync
